src/background/style-manager/matcher.js

```diff
@@ -1,80 +1,94 @@
-import {styleCodeEmpty} from '@/js/sections-util';
-import {ownRoot} from '@/js/urls';
-import {stringAsRegExpStr, tryRegExp, tryURL} from '@/js/util';
-import * as colorScheme from '../color-scheme';
+import { styleCodeEmpty } from "@/js/sections-util";
```

Comment:
Please undo all cosmetic changes (formatting/quotes/indents and so on) as it makes next to impossible to use git features like blame or history grep or `diff` as they become too "noisy" in almost any git client.

Reply:
Almost. There's still prefs.js and manage-newui.css.